ONSdigital/ras-frontstage
# Display suitable error messages to respondents us223

# Motivation and Context
Replaced error templates with new versions that do not present as errors to the user, removing technical 'jargon'.

# What has changed
Changed error templates
Added flash message to sign in page for when user is redirected after signing out

# How to test?
Run unit tests
Change timeout from 3600 to something like 30, after being timed out you will be redirected to the sign in page and an info panel should appear.

# Links
https://trello.com/c/EiwEvwud/811-display-suitable-error-messages-to-respondents-5

# Screenshots (if appropriate):
